openupgrade_scripts/scripts/event/17.0.1.8/pre-migration.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
```

Comment:
```suggestion
@openupgrade.migrate()
def migrate(env, version):
    fill_event_registration_company_name(env)
```
please, group them in a separated method